index.bs

```diff
@@ -1231,16 +1231,16 @@ interface MLGraphBuilder {
   // Create an operand for a graph constant.
   MLOperand constant(MLOperandDescriptor descriptor, ArrayBufferView bufferView);
 
-  // Create a single-value operand from the specified number of the specified type.
-  MLOperand constant(double value, optional MLOperandDataType type = "float32");
+  // Create a scalar operand from the specified number of the specified type.
+  MLOperand constant(MLOperandDataType type, double value);
```

Comment:
Should we use `MLNumber` for `value` parameter? I saw this usage in https://github.com/webmachinelearning/webnn/pull/647

Reply:
Eventually, but there's more discussion/design/prototyping needed over there.